releasenotes/notes/bosonic-operator-and-mapper-45bfde873f092681.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    Introduced a new feature that implements the bosonic operators ``BosonicOp``.
```

sphinx uses the following syntax to link class names to the corresponding API ref:

```suggestion
    Introduced a new feature that implements the bosonic operators :class:`.BosonicOp`.
```

---

Solved